01_Core/week02/submissions/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Cassidy's Webpage</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <div class="main">
+    <header id="home">
+        <div class="nav-bar">
+            <a href="#home" class="btn">Home</a>
```

Comment:
This is a list of links, so typically we use unordered lists for navs. It's more semantic.